doc/source/installation.rst

```diff
@@ -0,0 +1,33 @@
+************
+Installation
+************
+
+.. _quickstart:
+
+Qutip-Jax needs the development version of Qutip and installation is only possible from source
+
+.. code-block:: bash
+
+    pip install qutip --pre
+    pip install git+https://github.com/qutip/qutip-jax.git
+
+
+.. _prerequisites:
+
+Prerequisites
+=============
+This package use the development version of QuTiP, Jax, equinox and diffrax.
+
+The following to packages are used for plotting and testing:
+
+.. code-block:: bash
+
+    matplotlib pytest
+
+In addition
+
+.. code-block:: bash
+
+    sphinx numpydoc sphinx sphinx_rtd_theme
+
+are used to build and test the documentation.
```

Comment:
Minor thing: This perhaps  suggest that these packages need to be installed manually yet the testing requirements should be in setup.cfg and the documentation requirements and in doc/requirementes.txt.